main.html

```diff
@@ -0,0 +1,333 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PeachCSS</title>
+    <link rel="stylesheet" href="styles.css">  
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" />    
+    
+    
+</head>
+<body>
+    <nav class="navbar">
+        <div class="brand">
+            <div class="brand-title ">PeachCSS</div>
+        </div> 
+        <small>v0.0</small>
+  </nav>
+  <div class="wrapper">
+    <div class="sidebar">
+       <div class="sidebar-menu">
+        <a  class="indi-link" href="#Installation">Installation</a>
+        <a  class="indi-link" href="#Alert">Alert</a>
+        <a  class="indi-link" href="#Badge">Badge</a>
+        <a  class="indi-link" href="#Button">Button</a>
+        <a  class="indi-link" href="#Card">Card</a>
+        <a  class="indi-link" href="#Modal">Modal</a>
+        <a  class="indi-link" href="#Input">Input</a>
+        <a  class="indi-link" href="#List">List</a>
+        <a  class="indi-link" href="#Navigation">Navigation</a>
+        <a  class="indi-link" href="#Avatar">Avatar</a>
+        <a  class="indi-link" href="#Image">Image</a>
+        <a  class="indi-link" href="#Typography">Typography</a>
+        <a  class="indi-link" href="#Grid">Grid</a>
+       </div>        
+    </div>
+    <div class="main-content">
+        <div class="installation">
+            <h1 id="Installation" style="color: #CC3300;">Installation</h1>
+            <h2 style="color: cyan;">CSS</h2> 
+            <p style="color: green">Copy-paste the stylesheet link into head section of your web page before all other stylesheets to load our CSS & FontAwsome icons.</p>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28155%2C155%2C155%2C1%29&t=cobalt&wt=sharp&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fpeachcss.netlify.app%252Fstyles.css%2522%253E%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Fcdnjs.cloudflare.com%252Fajax%252Flibs%252Ffont-awesome%252F5.15.3%252Fcss%252Fall.min.css%2522%2520integrity%253D%2522sha512-iBBXm8fW90%252BnuLcSKlbmrPcLa0OT92xO1BIsZ%252BywDWZCvqsWgccV3gFoRBv0z%252B8dLJgyAHIhR35VZc2oM%252FgI1w%253D%253D%2522%2520crossorigin%253D%2522anonymous%2522%2520%252F%253E%2520%2520%2520%2520"
+                style="width: 784px; height: 240px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+        </div>
+        <div class="alerts">
+            <h1 id="Alert" style="color: #CC3300;">Alerts</h1>
```

Comment:
have a classname for each individual color you use, then you can use the same classname anywhere you want the same color